packages/mock/src/utils.ts

```diff
@@ -0,0 +1,46 @@
+import { Ref, KeyTypeConstraints } from './types';
+
+export function uuidv4() {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, c => {
+    const r = (Math.random() * 16) | 0;
+    // eslint-disable-next-line eqeqeq
+    const v = c == 'x' ? r : (r & 0x3) | 0x8;
+    return v.toString(16);
+  });
+}
+
+export const randomListLength = () => {
+  // Mocking has always returned list of length 2 by default
+  // return 1 + Math.round(Math.random() * 10)
+  return 2;
```

Comment:
Notice this: for backward compatibility, i make it return 2 but it would be richer if we'd truly return a random number.